src/lib/product_reviews/review/header.js

```diff
@@ -39,6 +46,7 @@ const Header = ({ rating, headline, date }) => {
             </StarsContainer>
             {headline}
          </Headline>
+         <ProductVariantName>{productName} | {productVariantName}</ProductVariantName>
```

Comment:
One suggestion here: maybe only show this if we have the values, so that it's compatible with the existing code that doesn't provide `productName`:
```suggestion
         {productName && productVariantName &&
          (<ProductVariantName>{productName} | {productVariantName}</ProductVariantName>)}
```

(I'm not sure I got the code formatting right on that; you might want to do it in your editor and run `prettier` to check.)

Reply:
Good idea! Fixed in 8c83ef1